pkg/apis/kubeone/types.go

```diff
@@ -208,6 +209,33 @@ type PodSecurityPolicy struct {
 	Enable bool `json:"enable"`
 }
 
+// StaticAuditLog feature flag
+type StaticAuditLog struct {
+	Enable bool                 `json:"enable"`
+	Config StaticAuditLogConfig `json:"config"`
+}
+
+// StaticAuditLogConfig config
+type StaticAuditLogConfig struct {
+	// PolicyFilePath is a path on local file system to the audit policy manifest
+	// which defines what events should be recorded and what data they should include.
+	// PolicyFilePath is a required field.
+	// More info: https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#audit-policy
+	PolicyFilePath string `json:"policyFilePath"`
+	// LogPath is path on control plane instances where audit log files are stored.
+	// Default value is /var/log/kubernetes/audit.log
+	LogPath string `json:"logPath"`
```

Comment:
if we hardcode policy path, why not LogPath?

Reply:
The reason why not hardcoding it is that I'm not sure will anyone need it in a special place, so it's possible to integrate with the desired tooling.